joomla/joomla-cms
# [4.0] Move custom fields from submenu to main menu in contact

### Summary of Changes
At the moment the custom fields are in the submenu for contact. With this PR The two submenu items are moved to the main menu like in the other extensions.

### Testing Instructions
- Apply patch
- Reinstall Joomla!
- Check menu items for contact
- disable/enable custom fields for contact to check display/hide
